modules/core/src/main/scala/lucuma/gen/gmos/longslit/GmosNorthLongSlit.scala

```diff
@@ -138,35 +134,31 @@ object GmosNorthLongSlit {
   object Science extends GmosNorthSequenceState {
 
     def compute(
-      mode:          ScienceConfigurationModel.Modes.GmosNorthLongSlit,
+      mode:          ScienceMode.GmosNorthLongSlit,
       exposureTime:  SciExposureTime,
       λ:             Wavelength,
       sourceProfile: SourceProfile,
       imageQuality:  ImageQuality,
       sampling:      PosDouble
-    ): ScienceSteps[NorthDynamic] = {
-
-      def sum(λ: Wavelength, Δ: Quantity[PosInt, Nanometer]): Wavelength =
-        new Wavelength(λ.toPicometers + Δ.to[PosInt, Picometer])
+    ): ScienceSteps[NorthDynamic] =
 
       eval {
         for {
           _  <- NorthDynamic.exposure      := exposureTime.value
-          _  <- NorthDynamic.xBin          := mode.fpu.xbin(sourceProfile, imageQuality, sampling)
-          _  <- NorthDynamic.yBin          := GmosYBinning.Two
+          _  <- NorthDynamic.xBin          := mode.xBin(sourceProfile, imageQuality, sampling)
+          _  <- NorthDynamic.yBin          := mode.yBin
```

Comment:
Slight use of advanced configuration here.  If you provide an explicit X or Y binning value it will get used.  The next step will be to use all the available advanced configuration options.